src/main/java/com/minecolonies/coremod/research/GlobalResearch.java

```diff
@@ -312,6 +317,12 @@ public int getDepth()
         return this.depth;
     }
 
+    @Override
+    public boolean isHidden()
```

Comment:
I love this possibility. I believe that to do this right though, we need to notify that a new research has been discovered, which means checking for these being unlocked onColonyTick somewhere. I'd suggest not on the university itself, but on the TownHall, so that it could be extended to auto-start researches marked appropriately there, and enable scenarios like blocking the university itself until some other criteria is met (School built?)